opencti-platform/opencti-graphql/src/modules/workspace/workspace.graphql

```diff
@@ -95,6 +95,7 @@ type Mutation {
     workspaceEditAuthorizedMembers(id: ID!, input:[MemberAccessInput!]!): Workspace @auth(for: [EXPLORE_EXUPDATE])
     workspaceContextPatch(id: ID!, input: EditContext!): Workspace
     workspaceContextClean(id: ID!): Workspace
+    configurationImport(file: Upload!): String!
```

Comment:
I wonder about the name of this mutation `configurationImport`, it doesn't explain well that it's a workspace import. In the code we declare it here in workspace.graphql but when we deploy, it won't be declared specifically for workspace domain (you can see it in graphql.ts, it's defined right after cityEdit), and we can use it directly through the API without knowing that it's linked to workspace.

I suggest `workspaceImport` what do you think ?

Reply:
`workspaceImport` implies the import of data and configuration.
We will use `workspaceConfigurationImport` to explicit the intent.